verify/cancel.py

```diff
@@ -0,0 +1,27 @@
+import os
+from os.path import join, dirname
+import sys
+from dotenv import load_dotenv
+
+dotenv_path = join(dirname(__file__), '../.env')
+load_dotenv(dotenv_path)
+
+NEXMO_API_KEY = os.getenv('NEXMO_API_KEY')
+NEXMO_API_SECRET = os.getenv('NEXMO_API_SECRET')
+
+if len(sys.argv) != 2:
+  print('Please supply request_id')
+  exit()
+
+REQUEST_ID = sys.argv[1]
+
+import nexmo
+
+client = nexmo.Client(key=NEXMO_API_KEY, secret=NEXMO_API_SECRET)
+
+response = client.cancel_verification(REQUEST_ID)
+
+if response['status'] == '0':
+  print('Cancellation successful')
```

Comment:
Please use 4-spaces per-tab. Only Google use 2-space indents, and we don't love them for it :)

Even better, just `pip install black` and then run black over your code, and it'll reformat all your code to the same format we use in `nexmo-python`.